llm-test/run_test.py

```
@@ -0,0 +1,180 @@
+        "options": {"temperature": 0.7, "top_p": 0.9},
+    }, ensure_ascii=False).encode("utf-8")
+
+    req = urllib.request.Request(
+        OLLAMA_URL,
+        data=body,
+        headers={"Content-Type": "application/json; charset=utf-8"},
+    )
+    try:
+        with urllib.request.urlopen(req, timeout=120) as resp:
+            return json.loads(resp.read().decode("utf-8"))
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)